client/src/components/UI/Modal.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Modal from '@material-ui/core/Modal';
+import Avatar from '@material-ui/core/Avatar';
+import Grid from '@material-ui/core/Grid';
+import ButtonGroup from './ButtonGroup';
+
+
+const styles = theme => ({
+  paper: {
+    position: 'absolute',
+    width: theme.spacing.unit * 100,
+    backgroundColor: theme.palette.background.paper,
+    boxShadow: theme.shadows[5],
+    padding: theme.spacing.unit * 4,
+    outline: 'none',
+    top: '40%',
+    left: '30%',
+  },
+  bigAvatar: {
+    margin: 10,
+    width: 100,
+    height: 100,
+  },
+});
+
+const style = {
```

Comment:
This can be added to paper class as both are applied to the same element.

Reply:
Ok I'll do that